Doc/c-api/type.rst

```diff
@@ -92,17 +92,8 @@ Type Objects
    from a type's base class.  Return ``0`` on success, or return ``-1`` and sets an
    exception on error.
 
-.. c:function:: PyObject* PyType_FromSpec(PyType_Spec *spec)
-
-   Creates and returns a heap type object from the *spec* passed to the function.
-
-.. c:function:: PyObject* PyType_FromSpecWithBases(PyType_Spec *spec, PyObject *bases)
-
-   Creates and returns a heap type object from the *spec*. In addition to that,
-   the created heap type contains all types contained by the *bases* tuple as base
-   types. This allows the caller to reference other heap types as base types.
-
-   .. versionadded:: 3.3
+   Note that :c:func:`PyType_FromSpecWithBases` calls ``PyType_Ready``
```

Comment:
Would not be better to document this in the description of `PyType_FromSpecWithBases()` instead?